etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/ChannelManager.scala

```diff
@@ -116,10 +121,18 @@ trait Authorization extends ChannelManager {
 
   private def extractToken(resp: Try[AuthenticateResponse]) = resp match {
     case Success(r) =>
-      val t = r.token
-      logger.info(s"ETCD: Got token $t")
+      val tokenStr = r.token
+      val t = Jwt.decode(tokenStr, JwtOptions(signature = false)) match {
+        case Success(claim) =>
+          logger.info(s"ETCD: Got jwt token $tokenStr")
```

Comment:
We shouldn't log credentials - tokenStr. 
It would be okay to info that we've updated token without the actual value.